wodles/aws/tests/test_aws.py

```diff
@@ -90,6 +91,52 @@ def test_metadata_version_services(mocked_db, class_):
         assert(metadata_version == ins.wazuh_version)
 
 
+@patch('os.path.exists', return_value=True)
+@patch('aws_s3.get_aws_config_params')
+@patch('aws_s3.debug')
+def test_default_config(mock_debug, mock_aws_config, mock_path_exists):
```

Comment:
The docstring format does not respect the standard.
Example:

https://github.com/wazuh/wazuh/blob/26364dcf3f5df6bb516cc0dae5fcd83399981004/wodles/aws/tests/test_aws.py#L321-L334